test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -1195,20 +1196,22 @@ def Xtest_cgroup_enforce_memsw(self):
         self.assertTrue("MemoryError" in tmp_out,
                         'MemoryError not present in output')
 
-    def Xtest_cgroup_offline_node(self):
+    def test_cgroup_offline_node(self):
         """
         Test to verify that the node is offlined when it can't clean up
         the cgroup and brought back online once the cgroup is cleaned up
         """
         name = 'CGROUP7'
         if 'freezer' not in self.paths:
             self.skipTest("Freezer cgroup is not mounted")
-        fdir = self.paths['freezer']
-        if not os.path.isdir(fdir):
+        fdir = self.get_cgroup_job_dir('freezer', '123.foo', self.momA)
+        fdir = os.path.dirname(fdir)
+        fdir = os.path.dirname(fdir)
```

Comment:
duplicate lines

Reply:
That was intentional.

Reply:
Can you turn these two comments into one that says "Get the parent's parent"